src/main/java/com/google/firebase/database/connection/WebsocketConnection.java

```diff
@@ -325,9 +325,9 @@ public void run() {
     @Override
     public void onError(final Throwable e) {
       if (e.getCause() != null && e.getCause() instanceof EOFException) {
-        logger.error("WebSocket reached EOF", e);
+        logger.debug("WebSocket reached EOF", e);
       } else {
-        logger.error("WebSocket error", e);
+        logger.debug("WebSocket error", e);
```

Comment:
Do you think it might make sense to still log all other errors at 'error' level? You should really only suppress the IllegalStateException you created in NettyWebsocketClient.

Reply:
Done